zeebe/gateway-protocol/src/main/proto/v2/batch-operations.yaml

```diff
@@ -609,6 +616,12 @@ components:
         - RESOLVE_INCIDENT
         - UPDATE_VARIABLE
 
+    BatchOperationActorTypeEnum:
+      type: string
```

Comment:
🔧 Let's add a description here for the enum. We might even move the description you added above to the attribute and simply use a simple ref at the attribute.

Reply:
@tmetzke I've copied this from `audit-logs.yaml` where the types don't have descriptions. If that's something we want to do, could we perhaps enforce it?

I'm happy to move the description (or copy it) for batch operations. Whatever you think is best. Let me know what you'd like me to do.

Reply:
I see, thanks, Nico 👍
We are missing too many descriptions in the spec right now to enforce it via linting. We first need to ramp up our spec files, unfortunately. They are currently just warnings that are not surfaced well in CI.

The best way to do this is to have descriptions for the types/schemas. No matter if objects, enums, or whatever. Then, when referencing them, ideally, you just use a `$ref` and that's it, the description is picked up from that object/enum/whatever. If you need to override it because your attribute/parameter needs to express something else, you can use the `allOf` + `description` at the attribute.